sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -224,12 +224,19 @@ def _build_ssl_opts(self, sslopts):
                 "SSL configuration must be a dictionary, or the value True."
             ) from None
 
-    def _build_ssl_context(
+    async def load_verify_locations(self, ctx, cafile=None):
+        if not cafile:
+            cafile = certifi.where()
+        loop = asyncio.get_event_loop()
+        with ThreadPoolExecutor() as pool:
```

Comment:
why run it in a custom threadpool when it can run in the default executor?

await loop.run_in_executor(None, ctx.load_verify_locations, cafile=cafile)

Reply:
gotcha, yep will update

Reply:
another thought - could this be moved to the `__init__` ?

Reply:
OOB: unfortunately, it can't be moved to the init b/c that would cause the error to raise in a different place, (object creation vs. open), and would be a breaking change.

Reply:
yes it should be. Trying it out.